tree_projections/tree_projections_utils.py

```diff
@@ -173,6 +177,12 @@ def relax_cond(mask, relax_mask, num_layers):
     batch_size = 256
     st = 0
 
+    if(diff):
```

Comment:
Does this work? Seems like there needs to be a space here.

Reply:
Neither did I lol. Fixed.

Reply:
XD